### ISSUE TITLE
fast5s not found within subfolders

### ISSUE THREAD

**Issue by scottgigante** [marcus1487/nanoraw#41]
It appears that nanoraw doesn't support the subdirectory structure adopted in ~April. Am I doing something wrong? My files are from the R9 era but have been manually sorted into subdirectories (1-19) in order to match the structure used by the latest versions of MinKNOW and Albacore.

```
nanoraw genome_resquiggle --bwa-mem-executable bwa --processes $(nproc) ecoli ecoli/Ecoli_K12_MG1655_ncbi.fa
Getting file list.
Correcting 19 files with 1 subgroup(s)/read(s) each (Will print a dot for each 100 files completed).

Failed reads summary:
        Error opening file. Likely a corrupted file. :  19
```

**Comment by marcus1487** [comment]
The recent sub directory structure is supported via the --fast5-pattern option. Nanoraw does not explicitly search recursive directories. Try running with '--fast5-pattern "*/*.fast5"'.

**Comment by scottgigante** [comment]
Much better, thanks! Made it work with `--fast5-pattern '*/*.fast5'` - didn't work without the asterisks.

**Comment by marcus1487** [comment]
Ah of course, responding from my phone, so formatting was tricky;)

I will add this to the error message and actually try to catch this even earlier. I think I might also add an explicit option to do this similar to the `--2d` option. Maybe `--recursive`? Is that misleading if I will only look one level deep?

**Comment by marcus1487** [comment]
Implemented `--recursive` in 9a5f4d1dce9d15207344ce45d75057d1ea8b141f

**Comment by scottgigante** [comment]
Brilliant, thanks. I don't think it's too misleading if you include that in the help text - unlikely that anyone will want to look deeper given current formats.
